src/no-result-found/style.scss

```diff
@@ -0,0 +1,6 @@
+.stc-sites-no-results {
+    border-bottom: 1px solid #D1D5DB;
```

Comment:
User CSS global variable instead of the static value. 

If it's something diff. then skip it.